tests/php/Controllers/Root/RootControllerTest.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+use App\Http\Controllers\Root\RootController;
+use Illuminate\Http\Request;
+
+/**
+ * RootControllerクラスのテストを行う.
+ * 
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class RootControllerTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)